kolibri/plugins/facility_management/assets/src/views/ClassEnrollForm.vue

```diff
@@ -194,7 +198,13 @@
   .actions-header,
   .footer,
   nav {
-    text-align: right;
+    text-align: end;
+  }
+  .pagination-button[dir='ltr'] {
```

Comment:
you might need to make separate rules for the "previous" and "next" buttons. The way this is, in RTL mode the "next" button (i.e. the one on the left in RTL) gets left margin and the "previous" button gets no right margin, so it is squished against the label.

Reply:
@jonboiser I was using hot reloading, which I suspect caused some of the issues where things worked out right on my end but not on yours.

Now that I did the dev using the normal dev server, I revised everything and it should be good to go.